src/main/java/uk/gov/ons/ctp/response/casesvc/service/CaseService.java

```diff
@@ -167,6 +169,86 @@ public Case findCaseByIac(final String iac) throws CTPException {
     return cazes;
   }
 
+  /**
+   * Find the paginated cases for a partyId ordered by create date descending.
+   *
+   * @param partyId the partyId
+   * @param maxCasesPerSurvey the maximum number of cases per survey to return
+   * @return the cases for the partyId
+   */
+  public List<Case> findCasesByPartyIdLimitedPerSurvey(
+      final UUID partyId, boolean iac, int maxCasesPerSurvey) {
+    log.info("Entering findCasesByPartyIdLimitedPerSurvey");
```

Comment:
As above

Reply:
as above

Reply:
As above (add context and it'll be useful!)

Reply:
As above!

Reply:
Agree  amman , changed to similar to that, but followed the existing convention of 'entering <func name>' . In this case its better than 'Finding cases by party id' since we have two methods of doing that, one not limited by survey , and the new one that does,